docs/devcontainer/README.md

```diff
@@ -7,6 +7,10 @@
 1. open your local clone of exago in VSCode
 1. copy the CoinHSL 2019 tarball to the top level of your exago directory
 
+## Generate the container
+
+Spack is used to generate the base image, with some sed/echo commands used to fine-tune Dockerfile for use case within a Dev Container. Run `.devcontainer/create_dockerfile.sh` in order to re-generate the Dockerfile. This should be done semi-frequently whenever a spack version or exago version needs updating.
```

Comment:
regenerate the Dockerfile? or that AND starts the build the build of said Dockerfile?

Reply:
This just created a file.

Build is with VSCode extension.

Script runs in like 3 seconds and just creates the entire Dockerfile from scratch